bundles/binding/org.openhab.binding.samsungac/src/main/java/org/openhab/binding/samsungac/internal/SamsungAcBinding.java

```diff
@@ -126,16 +124,18 @@ private String getCmdStringFromEnumValue(Command command, CommandEnum property)
         return cmd;
     }
 
-    private void sendCommand(AirConditioner aircon, CommandEnum property, String value) {
+    private void sendCommand(AirConditioner aircon, CommandEnum property, String value, String hostName) {
         int i = 1;
         boolean commandSent = false;
         while (i < 5 && !commandSent) {
             try {
                 logger.debug("[" + i + "/5] Sending command: " + value + " to property:" + property + " with ip:"
                         + aircon.getIpAddress());
-                if (aircon.sendCommand(property, value) != null) {
+                Map<CommandEnum, String> status = aircon.sendCommand(property, value);
+                if (status != null) {
                     commandSent = true;
                     logger.debug("Command[" + value + "] sent on try number " + i);
```

Comment:
Could you parameterize all the logging in the binding, in order to avoid lots of unnecessary object creation and garbage collection?